src/components/Utils/helpers.js

```diff
@@ -13,3 +14,10 @@ export const isProductDuplicated = (products, item) => {
   );
   return normalizedItemsDb.includes(normalizedItemInput);
 };
+
+export const isWithin24hours = (lastPurchasedDate) => {
+  const currentDate = +new Date();
+  const oneDay = 60 * 60 * 24 * 1000;
+  const isOutdated = currentDate - lastPurchasedDate > oneDay;
```

Comment:
you can `return currentDate - lastPurchasedDate > oneDay;` without having to assign it to a variable first, if you'd like

Reply:
We think in our way is more readable. Thanks for the advice. 😃